riker-macros/tests/macro.rs

```diff
@@ -95,3 +95,56 @@ fn run_derived_generic_actor() {
         std::thread::sleep(std::time::Duration::from_millis(50));
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct Message<T> {
+    inner: T,
+}
+
+#[actor(Message<String>)]
```

Comment:
There should also be a test to check if fully qualified types work. See example above.